Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/lists/List.tsx

```diff
@@ -49,6 +50,7 @@ const List = <TValue extends unknown>({
     wrap = false,
     fullWidth = false,
     scrollable = false,
+    itemsCount = 0,
```

Comment:
`0` is hacky. Make it `null` and check for `null`

Reply:
Also, taking a more detailed look at this - why are you changing the `List` to have `itesCount` to display? This seems to be logic that should be external to the `List`.
The `list`'s only job is to diplay a provided enumerable of values with a render function.
You have almost implemented this in `ContestFilter`, except the slicing.

@shsimeonova @mishogv thoughts?

Reply:
It is a great suggestion and I have implemented it.